code/modules/mob/living/carbon/human/species.dm

```diff
@@ -1140,14 +1140,16 @@
 /datum/species/proc/spec_hitby(atom/movable/AM, mob/living/carbon/human/H)
 	return
 
-/datum/species/proc/spec_attack_hand(mob/living/carbon/human/M, mob/living/carbon/human/H, datum/martial_art/attacker_style = M.martial_art)
+/datum/species/proc/spec_attack_hand(mob/living/carbon/human/M, mob/living/carbon/human/H, datum/martial_art/attacker_style)
 	if(!istype(M))
 		return
 	CHECK_DNA_AND_SPECIES(M)
 	CHECK_DNA_AND_SPECIES(H)
 
 	if(!istype(M)) //sanity check for drones.
 		return
+	if(H.mind)
```

Comment:
H > M. Could the arguments be removed if these are checked at this stage anyway ?